README.md

```diff
@@ -1,18 +1,281 @@
-puppet-module-tacacsplus
-========================
+# puppet-module-tacacsplus
 
-Puppet module to manage TACACS+ daemon from shrubbery.net
-Authenticates against PAM which uses VAS
+# Module description
 
-===
+Manage TACAS+.
 
 # Compatibility
 
-This module has been tested to work on the following systems with the
-latest version of Puppet 3 with the future parser enabled and Puppet v4
-with Ruby versions 2.1.0 and 2.3.1.
+This module is built for use with Puppet v3 (with and without the future
+parser) and Puppet v4 on the following platforms and supports Ruby versions
+1.8.7, 1.9.3, 2.0.0, 2.1.0 and 2.3.1.
 
  * EL 6
 
-===
+# Class Descriptions
+## Class `tacacsplus`
+### Parameters
 
+---
+#### acl (type: Hash)
+Containing names of Access Control Lists (ACL) and their permission and regex
+pairs to appear in the ACL section of tac_plus.conf. ACL rules will get sorted
+by their names. If sequence does matter, you could use ascending numbers as
+prefix for your rule names.
+
+Using the default of 'NONE' will not add any rules.
+
+- *Default*: 'NONE'
+
+##### Example:
+```yaml
+tacacsplus::acl:
+  'acl_all_features':
+    permit: '127.0.0.1'
```

Comment:
They show an hash ($acl) containing an array which contains hashes itself again. It's the array that allows multiple instances of the key value pairs.